docs/en_US/guides/sideloading-apps.md

```diff
@@ -52,24 +53,35 @@ Sideloadly is compatible with iOS 7 and newer.
 1. Plug your iOS device into your computer
     - Make sure your computer is trusted and allowed to view the contents of your device
 1. Drag and drop the `.ipa` file of your choice into Sideloadly
-1. Enter in your Apple ID
+1. Enter your Apple ID
 1. Enter in your password
-    - Sideloadly must make a request to it's servers in order to work with free developer accounts. If you are not OK with this, you may use an alternate Apple ID.
+    - Sideloadly must make a request to its servers in order to work with free developer accounts. If you are not OK with this, you may use an alternate Apple ID.
 
 ### Sideload using AltStore
 
-<!--I will add AltStore 1.5 soon don't worry-->
 
 ::: tip
 
-AltServer is only compatible with iOS 12.2 and newer.
+AltStore is only compatible with iOS 15 and newer.
```

Comment:
https://github.com/altstoreio/AltStore/blob/main/README.md says it's compatible with iOS 14+

It was changed on [October 20](https://github.com/altstoreio/AltStore/commit/c6227432e4cfea3869b825be8a69df835c6685f0). According to the [changelog](https://faq.altstore.io/release-notes/altstore), Altstore hasn't been updated since then, so it could be assumed that the latest version of Altstore still supports iOS 12.2+. Once it gets updated, I assume it will be possible to just download an older version anyway